USAGE_desktop_D3D12.md

```diff
@@ -183,7 +183,7 @@ Usage:
                         [--screenshot-dir <dir>] [--screenshot-prefix <file-prefix>]
                         [--sfa | --skip-failed-allocations] [--replace-shaders <dir>]
                         [--opcd | --omit-pipeline-cache-data] [--wsi <platform>]
-                        [--dcp | --discard-cached-psos] [--surface-index <N>]
+                        [--use-cached-psos <boolean>] [--surface-index <N>]
```

Comment:
I think it shouldn't be a boolean if it's off if you specify nothing but turned on when you specify it.  Making this a boolean would make it inconsistent with other command-line switches.